examples/zephyr/common/Kconfig

```diff
@@ -31,3 +135,30 @@ config GOLIOTH_SAMPLE_WIFI_PSK
 	  unsecure (open) AP.
 
 endif # GOLIOTH_SAMPLE_WIFI
+
+config GOLIOTH_SYSTEM_SETTINGS
```

Comment:
This should be something like `GOLIOTH_SAMPLE_SETTINGS` or `GOLIOTH_SAMPLE_PSK_SETTINGS`. There's a similar change pending in https://github.com/golioth/golioth-zephyr-sdk/pull/406.

Reply:
Sounds good, I've changed them to `GOLIOTH_SAMPLE_SETTINGS`.